idioms-json/file-and-directory-pattern.json

```diff
@@ -8,7 +8,7 @@
                 "ip-watchlist"
             ],
             "modified": "2014-05-08T09:00:00.000Z",
-            "pattern": "[(file:hashes.'SHA-256' = 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855' AND (file:file_name = 'iprip32.dll' AND file:parent_directory_ref.path = '/usr/local'))]",
```

Comment:
Do all of the file_name->name changes need to be in this branch? If not I'd rather keep them separate.